pac4j-saml/src/main/java/org/pac4j/saml/client/Saml2Client.java

```diff
@@ -17,47 +17,50 @@
 package org.pac4j.saml.client;
```

Comment:
We don't have any breaking change in this class, just internal ones: that's a great news!